e2e/credentials-server.e2e.spec.ts

```diff
@@ -305,12 +308,13 @@ describeWithCreds('Credentials server', () => {
     ({ stdout } = await rdctlCredWithStdin('get', bobsURL));
     expect(JSON.parse(stdout)).toMatchObject(body);
 
-    await expect(rdctlCredWithStdin('erase', bobsURL)).resolves.toMatchObject({ stdout: '' });
-
-    await expect(rdctlCredWithStdin('get', bobsURL)).rejects.toMatchObject({
-      stdout: expect.stringContaining('credentials not found in native keychain'),
-      stderr: expect.stringContaining('Error: exit status 22'),
-    });
+    if (credStore !== 'pass') {
```

Comment:
Same as above — needs a comment (or a description in `test.skip`).

Reply:
done